ps_checkout.php

```diff
@@ -1039,20 +1039,77 @@ public function hookDisplayAdminOrderMainBottom(array $params)
      */
     public function hookDisplayPaymentTop()
```

Comment:
I think it would be best to put hooks in classes to make the main class easier to read and understand. Over 1000 lines it's starting to do.